src/graphql/schema.ts

```diff
@@ -6,6 +6,7 @@ export const typeDefs = `#graphql
 
     type Query {
       hello: [SimpleText]
+      user(data: ID!): User!
```

Comment:
```suggestion
      user(id: ID!): User!
```

`data` dá a ideia de ser um objeto, mas teu parametro é só um campo